desktop-exporter/app/components/sidebar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { ArrowLeftIcon, ArrowRightIcon } from "@chakra-ui/icons";
+import { Flex, IconButton, useColorModeValue } from "@chakra-ui/react";
+
+import { TraceList } from "../components/traceList";
+import { TraceSummary } from "../types/api-types";
+
+type SidebarProps = {
+  isFullWidth: boolean;
+  toggle: () => void;
+  traceSummaries: TraceSummary[];
+};
+
+export function Sidebar(props: SidebarProps) {
+  const sidebarColour = useColorModeValue("pink.100", "pink.900");
```

Comment:
I appreciate your commitment to resisting American English in your naming

Reply:
Thank you. I do try ever so dutifully.